src/app/public/plugin-resources/code-examples/timepicker/basic/timepicker-demo.component.html

```diff
@@ -0,0 +1,42 @@
+<form
+  novalidate
+  [formGroup]="formGroup"
+>
+  <sky-timepicker
+    #timepicker
+  >
+    <input
```

Comment:
Could you make this 50% width and left-aligned to match the [design spec](https://blackbaud.invisionapp.com/share/T7RPR60C5NG#/screens/399164997_timepicker)? 

![Screen Shot 2020-09-15 at 2 05 50 PM](https://user-images.githubusercontent.com/18354182/93247697-b2616280-f75c-11ea-995b-90d31041dc58.png)